wagtail/admin/views/pages/listing.py

```diff
@@ -45,6 +49,17 @@ def get_filter_predicate(self, v):
         return {"path__startswith": v.root_page.path}
 
 
+class HasChildPagesFilter(BooleanFilter):
```

Comment:
This works well, although I wonder if we should also special-case `BooleanFilter` in `BaseListingView.active_filters` so instead of showing `True`/`False`, we show "Yes"/"No" like the form does.

e.g. by adding

```py
            elif isinstance(filter_def, BooleanFilter):
                filters.append(
                    ActiveFilter(
                        filter_def.label,
                        {True: _("Yes"), False: _("No"), None: _("Any")}[value],
                        self.get_url_without_filter_param(field_name),
                    )
                )
```

Or alternatively, using a `ChoiceFilter`/`TypedChoiceFilter` instead, so we don't have to write the "Yes"/"No" code twice.